src/constants.js

```diff
@@ -17,9 +17,9 @@ export const EthereumNetworkKeys = Object.freeze({
 
 // genesisHash is used as Network key for Substrate networks
 export const SubstrateNetworkKeys = Object.freeze({
-  // POLKADOT: '123',
   KUSAMA: '456',
```

Comment:
@Tbaut I'm fine with merging as is and coming back to update this, unless there are objections.

Reply:
right, we need several updated here and there for the readme, the T&Cs as well, so we can do that later.